src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1002,7 +1002,7 @@ def test_resource_policy_identity(self, resource_group, resource_group_location)
         self.cmd('policy assignment delete -n {pan} -g {rg}')
 
     @ResourceGroupPreparer(name_prefix='cli_test_policy_management_group')
-    @AllowLargeResponse()
+    @AllowLargeResponse(4096)
```

Comment:
Why do you want to specify object size for response here?

Reply:
When listing policies to ensure the modified policy is present it retrieves all built-in policies that Microsoft publishes.  This is a decent amount of content and was exceeding the default AllowLargeResponse size.  We are in the process of adding filters and better pagination to these APIs which should allow me to cut these response sizes down in the tests in a future change.